database/migrations/2021_08_20_082906_create_country_medals_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCountryMedalsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('country_medals', function (Blueprint $table) {
```

Comment:
@wahdam-amar unfortunately it's not a manytomany relationship that I described in the original video. Probably will still work, but in the upcoming result video I will show a better solution, in my opinion

Reply:
Thanks, yah you can show me the correct way in the coming video,ill learn from it